From 9b0a994d9804beea118753fb3c76542584fdcd17 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Fri, 14 Dec 2018 15:14:10 +0100
Subject: [PATCH] iproute2: Do not use libelf

Patch out libelf check in configure script as if libelf is by coincidence build
before iproute2, it gets selected which breaks dependencies and build.

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 .../utils/iproute2/patches/910-no-libelf.pach       | 13 +++++++++++++
 1 file changed, 13 insertions(+)
 create mode 100644 package/network/utils/iproute2/patches/910-no-libelf.pach

diff --git a/package/network/utils/iproute2/patches/910-no-libelf.pach b/package/network/utils/iproute2/patches/910-no-libelf.pach
new file mode 100644
index 0000000000..e2202790fe
--- /dev/null
+++ b/package/network/utils/iproute2/patches/910-no-libelf.pach
@@ -0,0 +1,13 @@
+Index: iproute2-4.19.0/configure
+===================================================================
+--- iproute2-4.19.0.orig/configure
++++ iproute2-4.19.0/configure
+@@ -252,7 +252,7 @@ EOF
+ 
+ check_elf()
+ {
+-    if ${PKG_CONFIG} libelf --exists
++    if false
+     then
+ 	echo "HAVE_ELF:=y" >>$CONFIG
+ 	echo "yes"
-- 
2.19.2

